docker/Makefile

```diff
@@ -0,0 +1,51 @@
+NAMESPACE ?= puppet
+git_describe := $(shell git describe)
+vcs_ref := $(shell git rev-parse HEAD)
+build_date := $(shell date -u +%FT%T)
+hadolint_available := $(shell hadolint --help > /dev/null 2>&1; echo $$?)
+hadolint_command := hadolint --ignore DL3008 --ignore DL3018 --ignore DL3028 --ignore DL4000 --ignore DL4001
+hadolint_container := hadolint/hadolint:latest
+pwd := $(shell pwd)
+export BUNDLE_PATH = $(pwd)/.bundle/gems
+export BUNDLE_BIN = $(pwd)/.bundle/bin
+export GEMFILE = $(pwd)/Gemfile
+
+version := $(shell echo $(git_describe) | sed 's/-.*//')
+dockerfile := Dockerfile
+
+prep:
+	@git fetch --unshallow 2> /dev/null ||:
+	@git fetch origin 'refs/tags/*:refs/tags/*'
+
+lint:
+ifeq ($(hadolint_available),0)
+	@$(hadolint_command) ezbake/$(dockerfile)
+else
+	@docker pull $(hadolint_container)
+	@docker run --rm -v $(PWD)/ezbake/$(dockerfile):/Dockerfile -i $(hadolint_container) $(hadolint_command) Dockerfile
+endif
+
+build: prep
+	  @docker build \
```

Comment:
(nitpick) This looks slightly misaligned with everything else

Reply:
updated